wgpu-core/Cargo.toml

```diff
@@ -29,6 +29,7 @@ targets = [
 
 [features]
 default = ["link"]
+api_log_info = []
```

Comment:
I know we have the bad habit in this repo not to, but can we document what the feature does here please?